packages/terra-search-field/src/SearchField.jsx

```diff
@@ -95,8 +104,10 @@ const propTypes = {
 const defaultProps = {
   defaultValue: undefined,
   disableAutoSearch: false,
+  groupName: 'Search',
```

Comment:
Should 'Search' be internationalized?

Reply:
Internationalization makes sense to me.
@eawww  What do you think?

Reply:
If it will be read by the screen reader or displayed to a user, it must be internationalized.

Reply:
This will be the default label name if isLabelVisible is set to true. Will work on internationalizing this.

Reply:
Logged a different Jira for internationalization. [UXPLATFORM-9454](https://jira2.cerner.com/browse/UXPLATFORM-9454)

Reply:
Even though you logged a JIRA for translations, you should still have the proper logic for i18n in this PR to use the intl object.  That should not be part of a separate PR.